source/Halibut/Diagnostics/LogCreators/AggregateLogWriterLogCreator.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using Halibut.Diagnostics.LogWriters;
+
+namespace Halibut.Diagnostics.LogCreators
+{
+    public class AggregateLogWriterLogCreator : ICreateNewILog
+    {
+        readonly ICreateNewILog nonCachingLogFactory;
```

Comment:
Nit: Do we care that this is 'non-caching'?

Could we just call it `createNewILog`? (or whatever name we came up with before in the previous comment)

Reply:
done